src/utils/mtev_log.h

```diff
@@ -171,6 +171,11 @@ API_EXPORT(int)
 #define SETUP_LOG(a, b) do { if(!a##_log) a##_log = mtev_log_stream_find(#a); \
                              if(!a##_log) { b; } } while(0)
 
+#define MTEV_LOG_AND_ABORT(log,args...) \
+  mtev_log_go_synch(); \
+  mtevL(log, args); \
+  abort();
+
```

Comment:
this macro should be wrapped in a `do {...} while(0)`, with no trailing semicolon... consider:
```
if(foo)
  MTEV_LOG_AND_ABORT(mtev_error, "blah");
```
which, with the current form, would only put `mtev_log_go_synch();` inside the `if` condition, and execute the next two statements unconditionally.